src/include/pmix_globals.c

```diff
@@ -154,6 +154,7 @@ static void ncdcon(pmix_nspace_caddy_t *p)
 static void ncddes(pmix_nspace_caddy_t *p)
 {
     if (NULL != p->ns) {
+        PMIX_RELEASE(p->ns->jobbkt);
```

Comment:
I'm not sure, but you may need to check for `p->ns->jobbkt` being `NULL` prior to trying to release it - probably safer to do so.